aiida/tools/importexport/dbexport/__init__.py

```diff
@@ -526,34 +806,34 @@ def export_tree(
             }},
             project=project_cols,
             tag=given_entity,
-            outerjoin=True
+            outerjoin=True,
         )
         entries_to_add[given_entity] = builder
 
-    # TODO (Spyros) To see better! Especially for functional licenses
-    # Check the licenses of exported data.
-    if allowed_licenses is not None or forbidden_licenses is not None:
-        builder = orm.QueryBuilder()
-        builder.append(
-            orm.Node, project=['id', 'attributes.source.license'], filters={'id': {
-                'in': node_ids_to_be_exported
-            }}
-        )
-        # Skip those nodes where the license is not set (this is the standard behavior with Django)
-        node_licenses = list((a, b) for [a, b] in builder.all() if b is not None)
-        check_licenses(node_licenses, allowed_licenses, forbidden_licenses)
+    return entries_to_add
+
+
+def _get_export_data(entries_queries: Dict[str, orm.QueryBuilder], silent: bool) -> Dict[str, Dict[int, dict]]:
```

Comment:
I'm a little bit confused as to what this function actually does - does it just collect queries or does it actually run them?

In any case, it would be useful to give a bit more expressive name, e.g.

```suggestion
def _perform_export_queries(entries_queries: Dict[str, orm.QueryBuilder], silent: bool) -> Dict[str, Dict[int, dict]]:
```

Reply:
No just collects the queries (again this was taken directly from the original code)

Reply:
changed name in b689f5e6986c15ab18ad225e5659f1ef3bf8f9c0